src/Courses/Settings/SettingsPage.tsx

```diff
@@ -0,0 +1,49 @@
+import { Drawer, Grid } from '@material-ui/core';
+import React, { useState } from 'react';
+import { Container, Row, Col } from 'react-bootstrap';
+import MaterialTriSelect from '../../Components/MaterialTriSelect';
+import { useCourseContext } from '../CourseProvider';
+import { OverridesForm } from './OverridesForm';
+import { CourseObject, UnitObject, UserObject, NewCourseTopicObj, ProblemObject, SettingsComponentType } from '../CourseInterfaces';
+
+import './SettingsPage.css';
+
+interface SettingsPageProps {
+
+}
+
+export const SettingsPage: React.FC<SettingsPageProps> = ({}) => {
+    const {course, setCourse, users, setUsers} = useCourseContext();
+    const [selected, setSelected] = useState<{
+        unit: UnitObject | undefined, topic: NewCourseTopicObj | undefined, problem: ProblemObject | undefined, user: UserObject | undefined
+    }>({unit: undefined, topic: undefined, problem: undefined, user: undefined});
+
+    return (
+        <Container>
+            <Row>
+                <Col className='text-center'>
+                    <h1>Course/Individual Settings</h1>
+                </Col>
+            </Row>
+            <MaterialTriSelect course={course} users={users} selected={selected} setSelected={setSelected} />
+            <Drawer 
+                className='black-drawer'
+                anchor='bottom' 
+                open={!!(selected.user && selected.topic)} 
```

Comment:
Nit: Double bang or Boolean wrapper 🤔 I think we differ on opinion here but I think wrapping in Boolean() is easier to read

Reply:
We do differ :joy: